tests/unit/dependencies.js

```diff
@@ -0,0 +1,167 @@
+(function () {
```

Comment:
It's a bit funny that you need these test because of the pattern of putting everything into a separate module, so it's kind of solving a problem we have created ourselves ;)